phpunit.xml

```diff
@@ -21,7 +22,8 @@
     <php>
         <env name="APP_ENV" value="testing"/>
         <env name="CACHE_DRIVER" value="array"/>
-        <env name="DB_DATABASE" value="parkitectnexus_test"/>
+         <env name="DATABASE" value=":memory:"/>
```

Comment:
Just create a local sqlite database in memory else Laravel complains. It not actually used since i end up mocking up the repositories.